app/src/main/res/layout/activity_board.xml

```diff
@@ -185,4 +185,27 @@
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@+id/boardLayout" />
 
+
+   <Button
```

Comment:
Remove this testing button

![image](https://github.com/gammaStrahlung/monopoly_app/assets/56844505/2f6b646c-dfee-4f55-98c6-086a0c5d1c8a)